2 dz/4/functions.h

```diff
@@ -0,0 +1,14 @@
+#pragma once
+#include <cmath>
+#include <iomanip>
+
+using namespace std;
+
+int gcd(int a, int b);
+void quicksort(double *row, int first, int last);
+struct Fraction
```

Comment:
I'm not insisting, but it might look a little better if there were blank lines between functions, structures, etc.